From e563ceb51b4c454d9d855e66ec92a31f8c4a493d Mon Sep 17 00:00:00 2001
From: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
Date: Sat, 14 Nov 2015 12:08:21 -0800
Subject: [PATCH] Makefile: customize the Python install bin directory

ustomize the Python bin directory to allow it to be installed within
the LUV native sysroot. Please note that the bin directory is set as BINDIR/PN.
This text is intended to be replaced at build time by bitbake with
the appropriate value.

Signed-off-by: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c5f94d4..b3ff0e4 100644
--- a/Makefile
+++ b/Makefile
@@ -174,7 +174,7 @@ dist: install-src-bits
 
 build-python-host: setup
 	$(Q)tar -cf - --exclude=.git -C $(BITS)/deps/python . | tar -xf - -C $(python-host-src)
-	$(Q)cd '$(python-host-src)' && ./configure
+	$(Q)cd '$(python-host-src)' && ./configure --bindir=BINDIR/PN --prefix=CUSTOM_PREFIX_DIR
 	$(Q)cd '$(python-host-src)' && $(MAKE)
 
 pylibtmp:=$(workdir)/python-lib
-- 
1.9.1

